### PR TITLE
Getting to where we think bitcoin should be.zh.md

### PR DESCRIPTION
**Pull request by Lezzkafka**
Chinese Trasnlate

### CODE REVIEW

**Comment by angryxueqiu** [comment]
@Lezzkafka 

Checked.
I love the title in chinese: "追赶比特币的理想旅途上" 
Good work.

I have some other pull request to be approved if you got spare time:
#380 #409 #412 #414 #419

**Comment by gz-c** [comment]
need sky address

**Comment by Lezzkafka** [comment]
@weihuakevin done :)

**Comment by weihuakevin** [comment]
approve

File: content/statement/Getting to Where We Think Bitcoin Should Be.zh.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
++++ title = "追赶比特币的理想旅途上" tags = [ "Decentralization", "Bitcoin", "Transaction Malleability", "Consensus", ] bounty = 4 date = "2017-10-03" categories = [ "Statement", ] +++ 这是来自2014年4月23日Bitcointalks的存档文章
```
Comment (gz-c) [inline content/statement/Getting to Where We Think Bitcoin Should Be.zh.md]: incorrrect formatting

**Comment by Lezzkafka** [comment]
@gz-c have a look :)

**Pull request merged**